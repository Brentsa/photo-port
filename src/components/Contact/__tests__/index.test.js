import React from 'react';
import { render, cleanup, getByTestId } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';
import ContactForm from "..";

afterEach(cleanup);

describe('Contact form renders', () => {
    it('Renders', () => {
        render(<ContactForm/>);
    })

    it('Matches snapshot', () => {
        const {asFragment} = render(<ContactForm/>);
        expect(asFragment).toMatchSnapshot();
    })

    it('Renders h1 and button', () => {
        const {getByTestId} = render(<ContactForm/>);

        expect(getByTestId('contact')).toHaveTextContent('Contact Me')
        expect(getByTestId('button')).toHaveTextContent('Submit')
    })
})