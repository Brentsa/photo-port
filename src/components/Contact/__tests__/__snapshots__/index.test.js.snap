// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact form renders Matches snapshot 1`] = `[Function]`;
